make.bat

```diff
@@ -73,6 +73,7 @@ if not defined SPHINXBUILD (
 	)
 	set PYTHON=venv\Scripts\python
 	set SPHINXBUILD=venv\Scripts\sphinx-build
+	set SPHIXAUTOBUILD=venv\Scripts\sphinx-autobuild
```

Comment:
Typo:
```suggestion
	set SPHINXAUTOBUILD=venv\Scripts\sphinx-autobuild
```
Or it might be worth updating all of them to use underscores to make them easier to read?